### PR TITLE
Add document

### PR DESCRIPTION
**Pull request by 0x26rew**
Please have a look.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# FPGA-DSP
+
+> 2022 Spring NCKU FPGA Course
+>
+> Homework 4
```
Comment (choucl) [inline README.md]: Homework 5

**Review by choucl — CHANGES_REQUESTED** [review]
It would be good to add a sequence diagram regarding to PS and controller

**Review by choucl — APPROVED** [review]
Great work, thanks!

**Review by jumha — APPROVED** [review]
Great!

**Pull request merged**